cmd/zfs/zfs_main.c

```diff
@@ -4577,6 +4581,13 @@ zfs_do_send(int argc, char **argv)
 		    resume_token));
 	}
 
+	if (flags.skipmissing && !flags.replicate) {
+		(void) fprintf(stderr,
+		    gettext("skip-missing flag can only be used in "
```

Comment:
Can it not be done in conjunction with -I as well? Or does that situation work without passing an additional flag?

Reply:
Honestly, that is not part of my workflow, neither was it suggested in the original PR. Would that be of great interest?

Reply:
I think it's reasonable to keep this restriction in place at least until we have a use case for it.